bot/exts/utils/reminders.py

```diff
@@ -348,8 +352,8 @@ async def list_reminders(self, ctx: Context) -> None:
             expiry = time.format_relative(remind_at)
 
             mentions = ", ".join([
-                # Both Role and User objects have the `name` attribute
-                mention.name async for mention in self.get_mentionables(mentions)
+                # Both Role and User objects have the `mention` attribute
+                mentionable.mention async for mentionable in self.get_mentionables(mentions)
```

Comment:
I just remembered, we don't usually put mentions in embeds as they often don't resolve (just show the ID in the mention rather than the name), so I'm not sure about this change.

Reply:
Fixed, as per my comment on Discord :+1: